### ISSUE TITLE
Bad 'help' for 'or' and 'not'

### ISSUE THREAD

**Issue by ToonTalk** [jmoenig/Snap#2595]
![image](https://user-images.githubusercontent.com/2138696/82929915-10ecb000-9fb7-11ea-9db4-3f5b83ce7d91.png)

There is no `play drum` block

![image](https://user-images.githubusercontent.com/2138696/82930008-3a0d4080-9fb7-11ea-81b0-5cd8e0c4322d.png)

`forever if` ??

But also shouldn't the help say what they do and just provide an example of their use? `and` isn't much better.

**Comment by brianharvey** [comment]
Yes, we should redo all of the help screens we inherited from Scratch.  Maybe someday.
